viral/ensemble_reactivation.py

```
@@ -0,0 +1,616 @@
+    return positions, speed, ITI_starts_ends, aligned_trial_frames
+
+
+def get_running_bouts(
+    place_cells: np.ndarray,
+    speed: np.ndarray,
+    frames_positions: np.ndarray,
+    aligned_trial_frames: np.ndarray,
+    config: GrosmarkConfig,
+) -> np.ndarray:
+    # TODO: think about the speed_threshold, arbitrarily set it to 5 cm/s
+    trial_activities = list()
```

Pylint (3.3.7)

[refactor] Consider using [] instead of list()

(R1734)